fspiop-api/documents/API-Definition_v1.1.1.md

```diff
@@ -2260,6 +2262,18 @@ The commit notification is sent in the request **PATCH /transfers/**_{ID}_ from
 
 **Figure 55 -- Commit notification where commit of transfer in Switch failed**
 
+In case of a validation failure (not schema validation) or another processing error during the processing of a **PUT /transfers/**_{ID}_ callback, a **PATCH /transfers/**_{ID}_**/error** notification is sent from the Switch to the Payee FSP to indicate of the failure. This notification is sent in both cases where the Payee FSP sends the _transferState_ as reserved or committed.
+
+[Figure 56](#figure-56) shows an example in which the validation in the Switch failed due to some reason, for example the a business rule validation caused a failure in the Switch. This is the same example as in [Figure 53](#figure-53), but where no reconciliation is needed as the Payee FSP receives a failure notification before performing the actual transfer to the Payee.
+
+###### Figure 56
+
+{% uml src="assets/diagrams/sequence/figure56.plantuml" %}
+{% enduml %}
+
+**Figure 56 -- Commit notification where commit of transfer in Switch failed**
```

Comment:
This is not a commit notification, it is an error notification for a commit failure in the Switch.